src/reactNodeToString.js

```diff
@@ -20,6 +20,8 @@ function reactArrayToJSXString (nodes) {
 export default function reactNodeToString (node) {
   if (Array.isArray(node)) {
     return reactArrayToJSXString(node)
+  } else if (typeof node !== 'object') {
```

Comment:
I'm not so sure about this check; should it maybe just be "if it's a string, return it"?

Reply:
I think I agree. @lelandrichardson suggested this way, but I think that having it throw when supplied say a number is better and more accurate then coercing it into a string.

Reply:
Is there a use case for allowing a non-string to work? It seems like that's likely to be a bug more than `.to.contain(3)` is used for convenience.